src/windows/View.tsx

```diff
@@ -60,6 +63,11 @@ export class View extends ViewCommon implements React.ChildContextProvider<ViewC
         ...ViewCommon.childContextTypes
     };
 
+    //Offset to show context menu using keyboard.
```

Comment:
See above. Add spaces after "//", fix name of method, and add spaces within curly brackets.

Reply:
updated as per conventions